### DISCUSSION TITLE
Tournament channels

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#77] [Fun Ideas]
Have a plugin to create tournament channels so there is no need to coordinate when we do the next table tennis tournament🤣

Ideally the plugin would create a channel, add participants and pair users based on a set of rules. Once finished the channel should be archived after a while (to let people congratulate winners and complain about how unfair everything was😅).

Also, doesn't this sound quite close to a playbook usecase? It would be great as that as well! It would put an emphasis in the play part of playbook

**Comment by larkox** [comment]
I love this idea for games, but somehow I feel this has to be useful for other more "serious" environment. No ideas coming to mind, but useful either way.

**Reply by Willyfrog** [reply]
for a tournament to make sense, it would need to fit on the organization running it, for mattermost I can only see it as part of our efforts to connect people together (like the table tennis one we recently had) and that in itself is seriuos enough
